pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -60,15 +60,6 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	// Watch for changes to secondary resource ComplianceRemediation and requeue the owner ComplianceSuite
-	err = c.Watch(&source.Kind{Type: &compv1alpha1.ComplianceRemediation{}}, &handler.EnqueueRequestForOwner{
-		IsController: true,
-		OwnerType:    &compv1alpha1.ComplianceSuite{},
```

Comment:
Wow, you're right. With this and our earlier discussion about removing the status completely, this is probably just dead code. But let me explain why I added it in the first place: to propagate the changes from the remediation to the suite status. So we can either go the way of removing the status or attempt to fix the watch. I guess fixing the watch might only make sense if there is a release between now and the time we remove the status and at the same time if we care about the status at all.

Reply:
I don't think we care much about the status. Especially not if we're trying to trim the status to be smaller. What that reconcile function will end up doing is aggregating the remediations and outputting a number... and the auto-apply.

Reply:
OK, cool.